packages/core/crates/core-crypto/src/types.rs

```diff
@@ -552,21 +558,25 @@ impl Display for OffchainPublicKey {
 }
 
 impl OffchainPublicKey {
```

Comment:
```suggestion
impl OffchainPublicKey {
    /// Generates a new random offchain public key.
    /// Because the corresponding private key is discarded, this might be useful only for testing purposes.
    pub fn random() -> Self {
        let (_, pk) = OffchainKeypair::random().unzip();
        pk
    }
```

Reply:
Also in `packages/hoprd/crates/hoprd-keypair/src/key_pair.rs` L.219, update to 
```
packet_key: ([0u8; PACKET_KEY_LENGTH], OffchainPublicKey::random()),
```

Reply:
fixed